### PR TITLE
Dinner recipe 2

### PR DESCRIPTION
**Pull request by yining-yan**
## 🎯 Objective

Add burger recipe to dinner

## 🛠️ Changes made

Added a new set of recipe in the same file

## 👥 Who needs to review this Pull Request? 
@Juanhuahua @anya-clowers

### CODE REVIEW

File: dinner/readme.md [hunk 1]
```diff
@@ -16,4 +16,33 @@
...
+Cheese – we love thick-sliced medium cheddar cheese for burgers, but you can use American-style cheese or your favorite sliced cheese.
+Buns – brioche buns taste great and have that steakhouse burger look, but any burger bun will work
+Lettuce – thinly slice your iceberg lettuce for a restaurant-style smash burger
+Pickles – sliced dill pickle chips are the best
+Tomatoes – Select meaty and large-sized tomatoes for burgers and cut them into rings
+Red onion – thinly sliced into rings
+
+
+## Equipment needed: 
+Pan, knife
```
Comment (Juanhuahua) [inline dinner/readme.md:39]: sounds perfect

File: dinner/readme.md [hunk 2]
```diff
@@ -16,4 +16,33 @@
...
+
+## Steps for preparation: 
+1) pan heat the beef 2) put the beef and other ingredients on buns 3)  Put it in the oven
+
```
Comment (Juanhuahua) [inline dinner/readme.md:43]: looks simple and clear

**Review by Juanhuahua — APPROVED** [review]
LGTM

**Review by anya-clowers — APPROVED** [review]
Looks good! Should we remove the pizza recipe?

**Review by Konkleffke — APPROVED** [review]
Nice work! Now we can choose what to cook 🍔

**Comment by Konkleffke** [comment]
> Looks good! Should we remove the pizza recipe?

I think we can leave it so everyone can choose what to eat

**Comment by anya-clowers** [comment]
> > Looks good! Should we remove the pizza recipe?
> 
> I think we can leave it so everyone can choose what to eat

Ok that works for me!

**Review by Juanhuahua — APPROVED** [review]

**Pull request merged**